src/components/space/charts-layout/ChartsArea.js

```diff
@@ -5,22 +5,10 @@ import ActionsByDayChart from '../charts/ActionsByDayChart';
 import ActionsMap from '../charts/ActionsMap';
 import ActionsByTimeOfDayChart from '../charts/ActionsByTimeOfDayChart';
 import ActionsByVerbChart from '../charts/ActionsByVerbChart';
-import { COMPOSE_VIEW_STRING } from '../../../config/constants';
-import ItemsByAccessedCount from '../charts/ItemsByAccessedCount';
 
 const ChartsArea = ({ actions, view, allUsers, usersToFilter }) => {
   return (
     <Grid container>
-      {view === COMPOSE_VIEW_STRING && (
-        <Grid item xs={12} sm={12} md={6} lg={6} xl={6}>
-          <ItemsByAccessedCount
-            actions={actions}
-            view={view}
-            allUsers={allUsers}
-            usersToFilter={usersToFilter}
-          />
-        </Grid>
-      )}
```

Comment:
Was there a possibility to show it for all views? I'm actually not sure what this chart was showing.

Reply:
This chart showed the most viewed items inside the space in the previous Graasp version (the analytics was shown only for the general space). Now, we do not have spaces, and everything is independent items (including folders). We could implement a chart that shows the most viewed items inside the item that is representing the analytics (we have to know which items are inside the parent item and retrieve its actions).

Reply:
I see, I guess we didn't handle getting actions for children items (in case of folders). Maybe let's comment the related code so we can reuse it when this will be implemented.